tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
+# We'll start this tutorial by loading one of the example datasets (called
+# ":ref:`sample-dataset`"), which contains EEG and MEG data from one subject
+# performing an audiovisual experiment, along with structural MRI scans for
+# that subject. The :func:`mne.datasets.sample.data_path` function will
+# automatically download the dataset if it isn't found in one of the expected
+# locations, then return the directory path to the dataset (see the
+# documentation of :func:`~mne.datasets.sample.data_path` for a list of places
+# it checks before downloading).
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, preload=False)
+
+###############################################################################
+# .. note::
 #
-# Access raw data
-# ^^^^^^^^^^^^^^^
-
-from mne.datasets import sample  # noqa
-data_path = sample.data_path()
-raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
-print(raw_fname)
-
-##############################################################################
-# .. note:: The MNE sample dataset should be downloaded automatically but be
-#           patient (approx. 2GB)
+#     :func:`~mne.io.read_raw_fif` takes a ``preload`` parameter, which
+#     determines whether the data will be copied into RAM or not. Some
+#     operations (such as filtering) require that the data be preloaded, but it
+#     is possible use ``preload=False`` and then copy raw data into memory
+#     later using the :meth:`~mne.io.Raw.load_data` method if needed.
+#
+# By default, :func:`~mne.io.read_raw_fif` displays some information about the
+# file it's loading; for example, here it tells us that there are three
+# "projection items" in the file along with the recorded data; those are
+# :term:`SSP projectors <projector>` for environmental noise, and are discussed
+# in a later tutorial. In addition to the information displayed during loading,
+# you can get a glimpse of the basic details of a :class:`~mne.io.Raw` object
+# by printing it:
 #
-# Read data from file:
+# .. TODO edit prev. paragraph when projectors tutorial is added: ...those are
+#     discussed in the tutorial :ref:`projectors-tutorial`. (or whatever link)
 
-raw = mne.io.read_raw_fif(raw_fname)
 print(raw)
-print(raw.info)
 
-##############################################################################
-# Look at the channels in raw:
+###############################################################################
+# :class:`~mne.io.Raw` objects also have several built-in plotting methods; in
+# interactive Python sessions the basic :meth:`~mne.io.Raw.plot` method is
+# interactive and allows scrolling, scaling, bad channel marking, annotation,
+# projector toggling, etc.
 
-print(raw.ch_names)
+raw.plot(duration=5, n_channels=30)
```

Comment:
perhaps also show raw.plot_psd and (maybe) raw.filter? No need for too much text ... the figure should speak for itself.